app/routes/admin/users/list.js

```diff
@@ -0,0 +1,98 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    let route = this.get('router.url').split('/');
+    return this.i18n.t(route[route.length - 1]);
```

Comment:
Dynamic strings won't be marked for translation properly. Use a switch case and return hardcoded strings like `this.i18n.t('Active')`